index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <section class="container">
+        <header class="nav-header">
+            <a href="index.html">
+                <img src="./media-files/Logo.png" alt="My Logo">
+            </a>
+            <nav>
+                <img src="./media-files/Enabled.png" alt="Menu">
+            </nav>
+        </header>
+        <article class="headline-main">
+            <h1 class="primary-text">
+                Hey There. <br>I'm Stephen.
+            </h1>
+            <h3 class="supporting-text">I'm a Software Developer</h3>
+            <p class="description">I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            <article class="socials">
+                <a href="index.html"><img src="./media-files/GitHub.png" alt="My-GitHub"></a>
+                <a href="index.html"><img src="./media-files/LinkedIn.png" alt="My-LinkedIn"></a>
+                <a href="index.html"><img src="./media-files/Reddit.png" alt="My-Reddit"></a>
+                <a href="index.html"><img src="./media-files/Twitter.png" alt="My-Twitter"></a>
+                <a href="index.html"><img src="./media-files/Medium.png" alt="My-Medium"></a>
+            </article>
+            <article class="group-img">
+                <img src="./media-files/Group-Image.png" alt="My-GitHub">
+            </article>
+        </article>
```

Comment:
- Great stuff 👍 but have you noticed it is not a major part of the content?

- [ ] As suggested, I think it's best to import this image using your `css`. It's just for the aesthetics, nothing seems to be so important about it such that wthout it, the content maybe viewed or understood differently